src/types.ts

```diff
@@ -38,6 +39,8 @@ export interface SliderComponentProps {
 type BreedsSpecs = [string, string | number | Length | ImageType][];
 
 export interface BreedDataProps {
+  fontSize?: string;
+  imgStyle?: string;
```

Comment:
I think it's better to describe imgStyle as https://www.typescriptlang.org/docs/handbook/release-notes/typescript-1-8.html#string-literal-types